ts/po_localizer.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import os
+
+PO_LINE_LEN = 80
+LOCALIZED_DIR = "localized"
+
+
+def localize_all_po_files(in_path, out_path, lang):
+    """Finds all .po files in a directory and localizes them in the given language.
+
+        Args:
+            in_path: The filepath for the directory with the .po files
+            out_path: (optional) The filepath to write the localized .po files to
+            lang: The target language (ISO-639-1 identifier) for localization
+    """
+    # TODO: Decide where to store & naming format for localized files
+
+    # Create directory for localized files to be placed in if not provided
+    if out_path is None:
+        out_path = os.path.join(in_path, LOCALIZED_DIR)
+        if not os.path.exists(out_path):
+            os.makedirs(out_path)
+
+    for f in os.listdir(in_path):
+        if f.endswith('.po'):
+            localize_po_file(in_path, out_path, f, lang)
+
+
+def localize_po_file(in_path, out_path, po_file, lang):
+    """Parses the given po file to find all msgid(untranslated string)
+        and add lang translation in corresponding msgstr(translated string)
+
```

Comment:
The msgid/msgstr thing is confusing if you don't know that's a PO file format thing. Maybe link to https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html or something?